### PR TITLE
Create 2017-02-17-django-ama.md

### PR DESCRIPTION
**Pull request by svbeon**

### CODE REVIEW

File: content/news/2017-02-17-django-ama.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+Title: Django AMA
+Author: Swant
+Date: 2017-02-17 17:55Z
+Slug: django-ama
+---
+
+Get your calendars out and start thinking of tricky questions,
+once again [the freenode community team](http://freenode.net/news/community)
+is pleased to announce another AMA, this time [Django](https://www.djangoproject.com)
```
Comment (nhandler) [inline content/news/2017-02-17-django-ama.md]: Write out what `AMA` stands for at least once.

File: content/news/2017-02-17-django-ama.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+will be our guest. Read more about them in the
+[blog post](http://freenode.net/news/django-and-freenode) published last week.
+
+Take the oppurtunity to ask Django all your questions about them,
```
Comment (nhandler) [inline content/news/2017-02-17-django-ama.md]: "Take the opportunity" sounds a bit odd. How about "This is your chance"?

File: content/news/2017-02-17-django-ama.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+Title: Django AMA
+Author: Swant
+Date: 2017-02-17 17:55Z
+Slug: django-ama
+---
+
+Get your calendars out and start thinking of tricky questions,
```
Comment (nhandler) [inline content/news/2017-02-17-django-ama.md]: This is a run-on sentence. Break it up into two sentences. I also understand the intent behind, "Get your calendars out" (you want them to make a note about when the session is), but starting off the post with this request feels odd. Focus on asking them to think up questions.

Comment (svbeon) [inline content/news/2017-02-17-django-ama.md]: Better?

**Comment by svbeon** [comment]
I'll see that as approved by someone native English, I hope @christeld won't mind?

**Pull request merged**